src/pages/trader-tools/_underlying-data.js

```diff
@@ -0,0 +1,444 @@
+import React from 'react'
+import {
+    INDEX10,
+    INDEX25,
+    INDEX50,
+    INDEX75,
+    INDEX100,
+    INDEX10S1,
+    INDEX100S1,
+    INDEX25S1,
+    INDEX50S1,
+    INDEX75S1,
+    RANGEBREAK100,
+    RANGEBREAK200,
+    STEPINDICES,
+    AUDCAD,
+    AUDCHF,
+    AUDNZD,
+    EURNZD,
+    GBPCAD,
+    GBPCHF,
+    GBPNZD,
+    NZDJPY,
+    GOLDUSD,
+    PALLADIUMUSD,
+    PLATINUMUSD,
+    SILVERUSD,
+    OILUSD,
+    AUDJPY,
+    AUDUSD,
+    EURCAD,
+    EURCHF,
+    EURJPY,
+    EURUSD,
+    GBPAUD,
+    GBPJPY,
+    GBPUSD,
+    USDCAD,
+    USDCHF,
+    USDJPY,
+    BOOM500,
+    BOOM1000,
+    CRASH500,
+    CRASH1000,
+    CHFJPY,
+    NZDUSD,
+} from 'components/elements/symbols.js'
+
+export const optionItemDefault = {
+    name: 'value',
+    displayName: 'Please select a value',
```

Comment:
we have to <Localize /> display name

and can change the variable name to display_name